projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -163,6 +163,26 @@ export default function PublicizeForm( {
 		[ isPublicizeDisabledBySitePlan, validationErrors ]
 	);
 
+	const renderInstagramNotice = () => {
+		return isEnhancedPublishingEnabled ? (
+			<Notice type={ 'warning' }>
+				{ __( 'You need a valid image or video in your post to share to Instagram.', 'jetpack' ) }
+				<br />
+				<ExternalLink href={ getRedirectUrl( 'jetpack-social-media-support-information' ) }>
+					{ __( 'Learn more', 'jetpack' ) }
+				</ExternalLink>
+			</Notice>
+		) : (
+			<Notice type={ 'warning' }>
+				{ __( 'You need a valid image in your post to share to Instagram.', 'jetpack' ) }
```

Comment:
I know this isn't strictly related to this change but there's something about the 'in your post' phrasing that doesn't seem quite right to me. It's more 'attached' but that doesn't seem to cover both the featured image and the attached media.

Like I mentioned in my other comment, without the advanced plan we could be specific and say that you need a featured image to share to Instagram.

Reply:
I changed it for the scenario when they do not have the advanced plan.

>I know this isn't strictly related to this change but there's something about the 'in your post' phrasing that doesn't seem quite right to me. It's more 'attached' but that doesn't seem to cover both the featured image and the attached media.

This is tricky, becuase they need either the featured image, attached image/video, or SIG turned on. I don't know if I can come up with a better and not longer explanation than the current one

Reply:
Yes, agreed, it's hard to make it succinct. What about something like:

"To share to Instagram, pick an image/video below, a featured image, or enable Social Image Generator"

or maybe just:

"To share to Instagram, add an image/video, or enable Social Image Generator"

Is there a way we could let them know in a tooltip that an image can be added in the media picker or as a featured image? In which case we could maybe make it:

"To share to Instagram, add an image or video to your post"

And lean on the fact that enabling Social Image Generator adds an image to the post.

I realise that 'add' is probably similarly confusing because it sounds like you can put an image in the post content, so we maybe need to be more explicit. but 'in' sounds like it's only in the post content to me.

We can probably go with something imperfect for now and adjust it later!

Reply:
> "To share to Instagram, add an image/video, or enable Social Image Generator"

I think this should do it, for anything more we should discuss it further in our call, or bring in design